src/commands/junit/api.ts

```diff
@@ -39,7 +39,7 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
     ...jUnitXML.spanTags,
     '_dd.cireport_version': '2',
   }
-  form.append('message', JSON.stringify(spanTags))
+  form.append('event', JSON.stringify(spanTags))
```

Comment:
this should make it work:
```suggestion
  form.append('event', JSON.stringify(spanTags), {filename: 'event.json'})
```
I don't know which is the file name we'd want there though

Reply:
Doesn't matter they won't upload it they are just validating